libs/natparse/src/main/java/org/amshove/natparse/parsing/CallLoopNode.java

```diff
@@ -0,0 +1,42 @@
+package org.amshove.natparse.parsing;
+
+import org.amshove.natparse.ReadOnlyList;
+import org.amshove.natparse.natural.ICallLoopNode;
+import org.amshove.natparse.natural.IOperandNode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class CallLoopNode extends StatementWithBodyNode implements ICallLoopNode
+{
+	private IOperandNode calling;
+	private final List<IOperandNode> operands = new ArrayList<>();
+
+	@Override
+	public IOperandNode calling()
+	{
+		return this.calling;
+	}
+
+	@Override
+	public ReadOnlyList<IOperandNode> operands()
+	{
+		return ReadOnlyList.from(operands);
+	}
+
+	void setCalling(IOperandNode calling)
+	{
+		this.calling = calling;
+	}
+
+	void addOperand(IOperandNode operandNode)
+	{
+		operands.add(operandNode);
+	}
+
+	@Override
+	public ReadOnlyList<IOperandNode> mutations()
+	{
+		return ReadOnlyList.from(operands);
```

Comment:
We need to only return operands that are variable references, because I'm getting the following diagnostics:

```
CALL 'CMZUL' 'R' #VAR #VAR2
```

`Operand is not modifiable by statement` on the string literal.

If `CONST` variables can't be passed to `CALL`, we can omit the `IMutateVariables` API for these statements